Model/User.php

```diff
@@ -170,24 +151,23 @@ public function unserialize($serialized)
     {
         $data = unserialize($serialized);
 
-        if (9 === count($data)) {
-            unset($data[4], $data[6]);
-
-            // add a few extra elements in the array to ensure that we have enough keys when unserializing
-            // older data which does not include all properties.
-            $data = array_merge($data, array_fill(0, 4, null));
+        if (13 === count($data)) {
```

Comment:
As this is done twice, maybe moving it into variable & re-use would be good idea?